tools/benchmark_all_runtimes.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
```

Comment:
I like to start my scripts like this, especially the "cd dirname", because it allows you to run the script from any folder and get the same result

```sh
# Exit on any error
set -e

# Always run the commands from the project dir
cd "$(dirname "$0")/.."
```

Reply:
Uuhh I like that! Committed

Reply:
Also the file is not executable, run `chmod +x script.sh` and commit

Reply:
Done